src/core/Akka.Persistence.Tests/ReceivePersistentActorAsyncAwaitSpec.cs

```diff
@@ -574,11 +574,13 @@ public AsyncPipeToDelayActor(string persistenceId)
 
                 CommandAsync<string>(async msg =>
                 {
+                    var sender = Sender;
+                    var self = Self;
                     Task.Run(() =>
                     {
                         Thread.Sleep(10);
                         return msg;
-                    }).PipeTo(Sender, Self); //LogicalContext is lost?!?
+                    }).PipeTo(sender, self); 
```

Comment:
Fixed this spec too - would sometimes fail because we didn't close over `Sender` .

Reply:
This isn't the final fix here - still having issues.